mqtt/mqtt-broker-tests-util/src/packet_stream.rs

```diff
@@ -35,19 +34,16 @@ impl PacketStream {
         let mut result = TcpStream::connect(&server_addr).await;
         let start_time = Instant::now();
         while result.is_err() {
-            tokio::time::delay_for(Duration::from_millis(100)).await;
+            tokio::time::sleep(Duration::from_millis(100)).await;
             if start_time.elapsed() > *DEFAULT_TIMEOUT {
                 break;
             }
             result = TcpStream::connect(&server_addr).await;
         }
 
         let tcp_stream = result.expect("unable to establish tcp connection");
-        let mut timeout = TimeoutStream::new(tcp_stream);
```

Comment:
There is this https://docs.rs/tokio/1.5.0/tokio/time/fn.timeout.html, not sure if it is reasonable to use here though... it also seems that `TimeoutStream` does not exist anymore from a cursory look into 1.5 docs

Reply:
@vadim-kovalyov what type of issue you anticipate `TimeoutStream` fixes? I couldn't reason a good one and just removed it for good

Reply:
Let's say in tests where we expect to receive a publication, but for whatever buggy reason broker did not send anything, it will fail fast instead of waiting till `keep_alive` or test timeout? Or am I missing anything?

Reply:
Hm... both server and client has the same standard value 5s but I understand we can change it potentially...